packages/wordpress-playground-block/src/components/playground-preview/index.tsx

```diff
@@ -296,158 +301,168 @@ export default function PlaygroundPreview({
-							{isNewFileModalOpen && (
-								<FileNameModal
-									title="Create new file"
-									onRequestClose={() =>
-										setNewFileModalOpen(false)
-									}
-									onSave={(newFileName) => {
-										addFile({
-											name: newFileName,
-											contents: '',
-										});
-										setActiveFileIndex(files.length);
-										setNewFileModalOpen(false);
-									}}
-								/>
-							)}
-						</div>
-						<div className="code-editor-wrapper">
-							<ReactCodeMirror
-								value={activeFile.contents}
-								extensions={[
-									keymapExtension,
-									EditorView.lineWrapping,
-									...getLanguageExtensions(
-										currentFileExtension || 'js'
-									),
-								]}
-								readOnly={codeEditorReadOnly}
-								onChange={(value) =>
-									updateFile((file) => ({
-										...file,
-										contents: value,
-									}))
-								}
-							/>
-						</div>
-						<div className="actions-bar">
-							{showFileControls ? (
-								<div className="file-actions">
-									{!activeFile && (
-										<button
-											type="button"
-											onClick={() => {
-												setEditFileNameModalOpen(true);
-											}}
-											className="wordpress-playground-block-button button-non-destructive"
-										>
-											<Icon icon={edit} /> Edit file name
-										</button>
-									)}
-									{!isErrorLogFile(activeFile) &&
-										files.filter(
-											(file) => !isErrorLogFile(file)
-										).length > 1 && (
-											<button
-												type="button"
-												className="wordpress-playground-block-button button-destructive"
-												onClick={() => {
-													setActiveFileIndex(0);
-													removeFile(activeFileIndex);
-												}}
-											>
-												<Icon
-													icon={cancelCircleFilled}
-												/>{' '}
-												Remove file
-											</button>
-										)}
-									{isEditFileNameModalOpen && (
-										<FileNameModal
-											title="Edit file name"
-											initialFilename={
-												files[activeFileIndex].name
-											}
-											onRequestClose={() =>
-												setEditFileNameModalOpen(false)
-											}
-											onSave={(fileName) => {
-												updateFile((file) => ({
-													...file,
-													name: fileName,
-												}));
-												setEditFileNameModalOpen(false);
-											}}
-										/>
-									)}
-								</div>
-							) : (
-								<div className="file-actions"></div>
-							)}
-							<Button
-								variant="primary"
-								icon="controls-play"
-								iconPosition="right"
-								onClick={() => {
-									handleReRunCode();
-								}}
-								className="wordpress-playground-run-button"
-							>
-								Run
-							</Button>
-						</div>
-					</div>
+				{!isActivated && (
+					<Button
+						className="wordpress-playground-activate-button"
+						onClick={() => setActivated(true)}
+					>Activate WordPress Playground</Button>
```

Comment:
```suggestion
					>Activate live preview</Button>
```

Reply:
Thanks, @adamziel! That describes the intent much better.

Should "live preview" be capitalized like "Activate Live Preview" or do we typically lowercase subsequent words on buttons?

Reply:
I like the capitalization! And let's also add aria attributes to communicate this will create an iframe with a full WordPress site embedded in it, and that it may be a challenge for screen readers.

Reply:
:+1: That sounds good.